File: CS/MyTabControl/MyXtraTabPage.cs
using DevExpress.XtraTab;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Example_HidePageHeader.MyTabControl {
    public class MyXtraTabPage : XtraTabPage {
        private bool _HidePageHeader;

        public bool HidePageHeader {
            get {
                return _HidePageHeader;
            }
            set {
                if(_HidePageHeader == value) {
                    return;
                }
                _HidePageHeader = value;
                OnPageChanged();
            }
        }
    }
}

File: VB/MyTabControl/MyXtraTabPage.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraTab
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace Example_HidePageHeader.MyTabControl
	Public Class MyXtraTabPage
		Inherits XtraTabPage
		Private _HidePageHeader As Boolean

		Public Property HidePageHeader() As Boolean
			Get
				Return _HidePageHeader
			End Get
			Set(ByVal value As Boolean)
				If _HidePageHeader = value Then
					Return
				End If
				_HidePageHeader = value
				OnPageChanged()
			End Set
		End Property
	End Class
End Namespace
